src/Binah/Frankie.hs

```diff
@@ -85,7 +86,7 @@ backend = getSqlBackend <$> getConfig
 {-@ assume httpAuthDb :: AuthMethod {v:(Entity User) | v == currentUser} (TaggedT<{\_ -> True}, {\_ -> False}> m)@-}
 httpAuthDb :: (MonadController w m, MonadConfig config m, HasSqlBackend config, MonadTIO m) => AuthMethod (Entity User) (TaggedT m)
 httpAuthDb = httpBasicAuth $ \username _password ->
-  mapTaggedT (reading backend) $ selectFirst (userNameField ==. username)
+  mapTaggedT (reading backend) $ selectFirst (EntityFieldWrapper UserName ==. username)
```

Comment:
I want to have a different name for the field wrapper. We can create a type class for this, but for now I'm fine with just requiring a particular name for the field (and not the wrapper).